R/generate_nowcasts.R

```diff
@@ -0,0 +1,85 @@
+#' Generate retrospective nowcasts
```

Comment:
This is a point nowcast right? Do we want t osay that clearly somewhere?

Reply:
Reworded in #50